common/lib/xmodule/xmodule/modulestore/split_mongo/__init__.py

```diff
@@ -23,6 +23,66 @@ def from_usage_key(cls, usage_key):
 CourseEnvelope = namedtuple('CourseEnvelope', 'course_key structure')
 
 
+class EditInfo(object):
+    """
+    Encapsulates the editing info of a block.
+    """
+    def __init__(self, edit_info={}):  # pylint: disable=dangerous-default-value
+        self.from_storable(edit_info)
+
+        # For details, see caching_descriptor_system.py get_subtree_edited_by/on.
+        self._subtree_edited_on = edit_info.get('_subtree_edited_on', None)
+        self._subtree_edited_by = edit_info.get('_subtree_edited_by', None)
+
+    def to_storable(self):
+        """
+        Serialize to a Mongo-storable format.
+        """
+        return {
+            'previous_version': self.previous_version,
+            'update_version': self.update_version,
+            'source_version': self.source_version,
+            'edited_on': self.edited_on,
+            'edited_by': self.edited_by,
+            'original_usage': self.original_usage,
+            'original_usage_version': self.original_usage_version,
+        }
+
+    def from_storable(self, edit_info):
```

Comment:
trivial / personal taste thing, but this could be implemented as a classmethod which could be used in concert with a more passive/straightforward `__init__` method that took kwargs or explicit args for everything.